app/scripts/modules/datepicker.js

```diff
@@ -109,7 +109,7 @@ angular.module('modified.datepicker', ['strap.position'])
             yearRange = getValue($attrs.yearRange, dtConfig.yearRange);
 
         this.minDate = dtConfig.minDate ? new Date(dtConfig.minDate) : null;
-        this.maxDate = dtConfig.maxDate ? new Date(dtConfig.maxDate) : null;
+        this.maxDate = null;
```

Comment:
You are not supposed to make this null, please make the maxDate declaration in the config section null:
https://github.com/aks681/community-app/blob/4fed2e5d75d65aa3d65e93a55f884c60f4b479ad/app/scripts/modules/datepicker.js/#L96